opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -340,7 +311,16 @@ class WellModelGhostLastMatrixAdapter : public Dune::AssembledLinearOperator<M,X
               iterations_( 0 ),
               converged_(false)
         {
+#if HAVE_MPI
+            comm_.reset( new communication_type( simulator_.vanguard().grid().comm() ) );
+#endif
             parameters_.template init<TypeTag>();
+            useFlexible_ = parameters_.use_cpr_ || EWOMS_PARAM_IS_SET(TypeTag, std::string, LinearSolverConfiguration);
```

Comment:
I think we should eventually retire the --use-cpr parameter, in favour of --linear-solver-configuration, for now this is fine I think.

Reply:
Leaving this for another PR